pcsx2/gui/RecentIsoList.cpp

```diff
@@ -178,6 +178,17 @@ void RecentIsoManager::InsertIntoMenu( int id )
 
 	if ( !exists )
 		curitem.ItemPtr->Enable( false );
+	else
+		curitem.ItemPtr->Enable( !g_Conf->AskOnBoot );
```

Comment:
Combine the if and else? `curitem.ItemPtr->Enable(exists && !g_Conf->AskOnBoot);`

Apart from that, I think it's fine.